### PR TITLE
Feature/tao 4279 export exclude empty

### PR DESCRIPTION
**Pull request by ssipasseuth**
This PR improves the QTI Item export form to disabled invalid item to be exported and show a tool tip explaining the reason why.
To test this:
1. create a new item class,
2. create a few items and leave at least one empty
3. open the item class export form -> check the result
4. open individual item form -> check the result

requires https://github.com/oat-sa/extension-tao-item/pull/270 to add the tooltips for disabled items

### CODE REVIEW

File: model/Export/ExportForm.php [hunk 1]
```diff
@@ -127,17 +141,38 @@ public function initElements()
...
-			$instanceElt->setValue($value);
+			if(!isset($disabledOptions[$value])){
+				$instanceElt->setValue($value);
+			}
 		}
 		$this->form->addElement($instanceElt);
 
 
-    	$this->form->createGroup('options', __('Export QTI 2.1 Package'), array('filename', 'instances'));
+    	$this->form->createGroup('options', $this->getFormGroupName(), array('filename', 'instances'));
```
Comment (eacraft13) [inline model/Export/ExportForm.php]: This should be updated depending on the QTI version.
screen shot 2017-05-12 at 2 18 56 am

**Comment by ssipasseuth** [comment]
fixed the form name : please check again

File: manifest.php [hunk 2]
```diff
@@ -29,7 +29,7 @@
     'label'       => 'QTI item model',
     'description' => 'TAO QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '8.3.0',
+    'version'     => '8.4.0',
     'author'      => 'Open Assessment Technologies',
     'requires' => array(
         'taoItems' => '>=4.1.0',
```
Comment (eacraft13) [inline manifest.php]: Requires should be updated for tao version 10.2.0

**Comment by ssipasseuth** [comment]
ok all fixed

**Pull request merged**